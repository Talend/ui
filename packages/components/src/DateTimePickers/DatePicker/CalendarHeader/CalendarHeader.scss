
.container {
    display: flex;
    direction: column;

    .day-of-week {
        padding: 10px;
        width: 2.5em;
        text-align: center;
    }
}
