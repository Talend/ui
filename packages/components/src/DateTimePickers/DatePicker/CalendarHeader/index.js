import CalendarHeader from './CalendarHeader.component';

export default CalendarHeader;
