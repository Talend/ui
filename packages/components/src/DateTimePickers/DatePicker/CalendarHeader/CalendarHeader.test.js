import React from 'react';
import { shallow } from 'enzyme';

import CalendarHeader from './CalendarHeader.component';

describe('CalendarHeader', () => {
	it('should render a CalendarHeader', () => {
		// when
		const wrapper = shallow(<CalendarHeader />);

		// then
		expect(wrapper.getElement()).toMatchSnapshot();
	});
});
