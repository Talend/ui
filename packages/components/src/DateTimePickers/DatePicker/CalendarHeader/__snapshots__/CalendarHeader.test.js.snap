// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalendarHeader should render a CalendarHeader 1`] = `
<div
  className="theme-container"
>
  <abbr
    className="theme-day-of-week"
  >
    L
  </abbr>
  <abbr
    className="theme-day-of-week"
  >
    M
  </abbr>
  <abbr
    className="theme-day-of-week"
  >
    M
  </abbr>
  <abbr
    className="theme-day-of-week"
  >
    J
  </abbr>
  <abbr
    className="theme-day-of-week"
  >
    V
  </abbr>
  <abbr
    className="theme-day-of-week"
  >
    S
  </abbr>
  <abbr
    className="theme-day-of-week"
  >
    D
  </abbr>
</div>
`;
